config/kubernetes-sigs/sig-api-machinery/teams.yaml

```diff
@@ -59,27 +59,15 @@ teams:
   controller-tools-admins:
     description: admin access to controller-tools
     members:
-    - DirectXMan12
-    - droot
-    - pwittrock
-    - seans3
-    privacy: closed
-  controller-tools-contributors:
-    description: read access to controller-tools
-    members:
-    - Birdrock
-    - pwittrock
+    - joelanford
+    - vincepri
     privacy: closed
   controller-tools-maintainers:
     description: write access to controller-tools
     members:
     - alvaroaleman
-    - camilamacedo86
-    - droot
     - joelanford
-    - mengqiy
```

Comment:
@mengqiy 👋 seems like Solly added you [in this commit](https://github.com/kubernetes-sigs/controller-tools/commit/0662bea6af62ef389c27a543381bdf5175235ab6) which was made alongside this other [commit in controller-runtime](https://github.com/kubernetes-sigs/controller-runtime/commit/71e9a1626803390e880d668142f062b613572956). Since then @mengqiy was moved to emeritus after inactivity in this [controller-runtime commit](https://github.com/kubernetes-sigs/controller-runtime/commit/344e96e1accc8bed25d09bfd8bd935bb44943081)

Reply:
The above seems also true for @droot